content/hi/docs/tutorials/configuration/configure-redis-using-configmap.md

```diff
@@ -0,0 +1,253 @@
+---
+reviewers:
+- eparis
+- pmorie
+title: कॉन्फ़िगमैप का उपयोग करके रेडिस को कॉन्फ़िगर करना
+content_type: ट्यूटोरियल
```

Comment:
The front matter `content` value should remain unchanged; there's no need to localize it.

```suggestion
content_type: tutorial
```